vscode/quint-vscode/server/src/complete.ts

```diff
@@ -194,33 +229,48 @@ function builtinCompletionsWithDocs(
 }
 
 function getSuggestedBuiltinsForType(type: QuintType): { name: string }[] {
+  logger.info(`🔍 Checking builtins for type: ${type}`)
```

Comment:
This prints `[object Object]` currently
```suggestion
  logger.info(`🔍 Checking builtins for type: ${type.kind}`)
```